src/utils.js

```diff
@@ -0,0 +1,80 @@
+/*
+ Copyright 2019 Square Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+function hasNonNullProperty(obj, property) {
+  if (Object.prototype.hasOwnProperty.call(obj, property) && typeof obj[property] !== 'undefined' && obj[property] !== null) {
+    return true;
+  }
+  return false;
+}
+
+export default function ValidateCheckoutParameters(checkoutParams) {
```

Comment:
can checkoutParams be null or undefined?

Reply:
Added addition validation for that.